### DISCUSSION TITLE
[email] Error: 7036:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:332:

### DISCUSSION THREAD

**Discussion by ahmed-abess** [directus/directus#13228] [Archive]
### Describe the Bug

Hello
when i try to send reset password request with directus i get this message
[email] Error: 7036:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:332:
note: i use directus 9 under windows 11

### To Reproduce

email config
EMAIL_FROM="no-replay@gmail.com"
EMAIL_TRANSPORT="smtp"

# ## Email (SMTP Transport)
# EMAIL_SMTP_POOL=true
EMAIL_SMTP_HOST="smtp.gmail.com"
EMAIL_SMTP_PORT=587
# EMAIL_SMTP_IGNORE_TLS=false
EMAIL_SMTP_USER="*************"
EMAIL_SMTP_PASSWORD="*******"
EMAIL_SMTP_SECURE=true # Use TLS

### Errors Shown

 [email] Error: 14008:error:1408F10B:SSL routines:ssl3_get_record:wrong version number:c:\ws\deps\openssl\openssl\ssl\record\ssl3_record.c:332:

### What version of Directus are you using?

9.9.1

### What version of Node.js are you using?

14.17.5

### What database are you using?

mysql

### What browser are you using?

chrome

### How are you deploying Directus?

locally windows

**Comment by azrikahar** [comment]
This seems similar to #11236. Can you try setting `EMAIL_SMTP_SECURE=false` and see how it goes? Also this comment may be relevant here: https://github.com/directus/directus/discussions/11236#discussioncomment-2034589

**Reply by ahmed-abess** [reply]
thx i solve it it was a problem of port i juste change it to 465 now it s ok

**Comment by promisingdev72** [comment]
Hello, I am using Outlook mail for sending email.
![image](https://user-images.githubusercontent.com/80725175/182687341-6a112d81-c25a-4de4-9ff5-56d1f8878a30.png)
But it shows me this issue.
![image](https://user-images.githubusercontent.com/80725175/182687163-a6e66c72-86f8-4a9e-afe7-e9e4a23e5760.png)

**Reply by rijkvanzanten** [reply]
Do you have any corporate firewalls or other network configuration that might prevent outbound communication to port 587?

**Reply by promisingdev72** [reply]
no, I am running it in my local pc now.

**Reply by promisingdev72** [reply]
![image](https://user-images.githubusercontent.com/80725175/182689326-94345054-cc25-4c2a-b592-595191f12db0.png)
When I use 587 port, Here is issue.

**Reply by rijkvanzanten** [reply]
Ah, I'd say you've find your problem: `Authentication unsuccessful, SmtpClientAuthentication is disabled for Tenant`. Looks like your Outlook account doesn't allow SMTP use 👍🏻

**Reply by promisingdev72** [reply]
Can you let me know how can i fix this issue?

**Reply by rijkvanzanten** [reply]
I have no clue, not a user of Outlook myself. 

> Visit https://aka.ms/smtp_auth_disabled for more information.

It looks like the error even includes a link to the outlook docs, so I'd start there 😉

**Reply by promisingdev72** [reply]
Thank you for your kind response.
😊
